tools/c7n_org/c7n_org/cli.py

```diff
@@ -171,14 +171,15 @@ def report_account(account, region, policies_config, output_path, debug):
 @click.option('-s', '--output-dir', required=True, type=click.Path())
 @click.option('-a', '--accounts', multiple=True, default=None)
 @click.option('--field', multiple=True)
+@click.option('--no-default-fields', default=False, is_flag=True)
 @click.option('-t', '--tags', multiple=True, default=None)
 @click.option('-r', '--region', default=['us-east-1', 'us-west-2'], multiple=True)
 @click.option('--debug', default=False, is_flag=True)
 @click.option('-v', '--verbose', default=False, help="Verbose", is_flag=True)
 @click.option('-p', '--policy', multiple=True)
 @click.option('--format', default='csv', type=click.Choice(['csv', 'json']))
 @click.option('--resource', default=None)
-def report(config, output, use, output_dir, accounts, field, tags, region, debug, verbose, policy, format, resource):
+def report(config, output, use, output_dir, accounts, field, no-default-fields, tags, region, debug, verbose, policy, format, resource):
```

Comment:
not quite.. click automatically transforms `-` into `_` to be a valid python variable, so the exposed cli flag is --no-default-fields but the variable passed to the function is no_default_fields ..

Reply:
oh ok I get it, that makes sense now.  Thanks